.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.json]
+indent_style = space
```

Comment:
You don't have tabs here.. that's why: https://github.com/LeaVerou/bliss/blob/gh-pages/package.json